RELEASE-NOTES.txt

```diff
@@ -1,15 +1,16 @@
-Welcome to the 0.9.4 open source release of the Simulation Engine! SimEng is a fast, easy to use and modify, cycle-level simulator for CPUs. Its initial focus is on simulating single Arm cores in server CPUs, and so the instruction set architecture (ISA) target is initially Armv9.2-a+SVE. Later versions of the Arm ISA, and other ISAs such as RISC-V, will be supported in future releases.
+Welcome to the 0.9.5 open source release of the Simulation Engine! SimEng is a fast, easy to use and modify, cycle-level simulator for CPUs. Its initial focus is on simulating single Arm cores in server CPUs, and so the instruction set architecture (ISA) target is initially Armv9.2-a+SVE. Later versions of the Arm ISA, and other ISAs such as RISC-V, will be supported in future releases.
```

Comment:
Arm9.2-a+SVE+SME*
Also we now support RISC-V (RV64) but this paragraph says we don't

Reply:
Added some stuff for RISC-V